### PR TITLE
Cycle 6 release

### PR DESCRIPTION
**Pull request by kleber-amido**
Cycle 6 has been released and new features have been added.
These changes here reflect these main changes, being:

- Web API for .Net and Java
- Web API with CQRS for .Net and Java (moved)
- Web API with CQRS and Events for .Net and Java
- CLI is being redeveloped (cycle 7)
- Addition of templates for .Net and Maven scaffolding for Java

### CODE REVIEW

**Review by radoslaw-orlowski — APPROVED** [review]

File: docs/workloads/azure/backend/netcore/architecture/project_structure_netcore.md [hunk 1]
```diff
@@ -42,6 +54,21 @@ Solution 'xxAMIDOxx.xxSTACKSss.API'
    └── xxAMIDOxx.xxSTACKSxx.Infrastructure.IntegrationTests
 ```
 
+<p><b>Functions and Worker Services</b></p>
+
+```text
+├── Functions
```
Comment (underscoreHao) [inline docs/workloads/azure/backend/netcore/architecture/project_structure_netcore.md:60]: This is no longer correct

Comment (underscoreHao) [reply docs/workloads/azure/backend/netcore/architecture/project_structure_netcore.md:60]: We'll fix that when updating the .NET information

**Review by underscoreHao — APPROVED** [review]

**Pull request merged**